diplo.lua

```diff
@@ -0,0 +1,88 @@
+--[[
+diplo - Quick "Diplomacy"
+Without arguments:
+    Print list of civs the player civ has diplomatic relations to.
+    This list might not match the ingame civ list, but is what matters. Does not include 'no contact' civs, etc.
+    Also shows if relations match between player civ and other civ - a mismatch can potentially happen 'naturally', but is important when trying to make peace proper, as both civs need to be at peace with the other.
+With arguments:
+    diplo CIV_ID RELATION

```

Comment:
consider allowing the keyword `all` instead of a `<CIV_ID>` so players can choose harmony or king of the hill style gameplay

Reply:
I might add that, though I think that'd need some more convoluted code. Probably need a more or less static listing of civ ids to iterate over for 'all'.

Reply:
I thought it would be a loop similar to the current listing function. is there more to it than thtat?

Reply:
Well, basically it is - but just copying the loop with slight changes feels redundant. Might be better to wrap more things in functions and such. I'll see how it goes when I get back to it.

Reply:
`all` argument can now be used, looping over the civ_list for civ ids.